src/Cafe/OS/libs/nn_olv/nn_olv.cpp

```diff
@@ -198,10 +201,35 @@ namespace nn
 			osLib_returnFromFunction(hCPU, memory_getVirtualOffsetFromPointer(&portalAppParam->serviceToken));
 		}
 
-		uint32 UploadPostDataByPostApp(void *postParam)
+		static SysAllocator<OSThread_t> s_OlvReleaseBgThread;
+		SysAllocator<uint8, 1024 * 128> s_OlvReleaseBgThreadStack;
```

Comment:
Since the OlvRelease thread doesn't really need any stack, I recommend changing the size from ``1024 * 128` to just `1024`.